### ISSUE TITLE
Tests: Setup automatic tests with github workflow for this project.

### ISSUE THREAD

**Issue by amarts** [cord-network/cord.js#24]
So that we are absolutely sure while merging that the code changes are not causing a regression

**Comment by Gmin2** [comment]
We need to add a github actions to trigger when saomeone create a pr for this repo right

**Comment by amarts** [comment]
> We need to add a github actions to trigger when saomeone create a pr for this repo right

Yes. There is already a check for `yarn build` (ie, the build is successful) but need is to add more tests (like running demo scripts etc).
